source/designer/source/DocumentCanvas.cpp

```diff
@@ -67,7 +67,7 @@ void DocumentCanvas::SelectControls( ControlList& CtrlList )
 {
 	m_selectionLayer->ClearSelection();
 
-	for ( ControlList::List::const_iterator it = CtrlList.list.begin(); it != CtrlList.list.end(); ++it )
+	for ( ControlList::List::const_iterator it = CtrlList.list.cbegin(); it != CtrlList.list.cend(); ++it )
```

Comment:
I was going to replace them with range for, but wanted to check if that was what you intended first. I noticed the code base was sticking to a bit more of a C++98 style, despite billing itself as C++11.